contracts/facade/facets/ReadFacet.sol

```diff
@@ -21,56 +22,11 @@ import "../../p1/StRSRVotes.sol";
  * @custom:static-call - Use ethers callStatic() to get result after update; do not execute
  */
 // slither-disable-start
-contract ReadFacet is IReadFacet {
+contract ReadFacet is MaxIssuableFacet, IReadFacet {
```

Comment:
This is not necessary, each facet can be it's own thing.

Reply:
it's nice for testing at the moment. it means we can just test the ReadFacet and not have to track a bunch of individual facets